server/server.ts

```diff
@@ -4,11 +4,14 @@ import genAI from '@server/routes/genAi';
 import { globalErrorHandler } from '@server/middleware/errorHandling';
 import bodyParser from 'body-parser';
 import { MongoClient, ServerApiVersion } from 'mongodb';
+import mongoose from 'mongoose';
 import type { Express, Request, Response } from 'express';
 import type { Server } from 'http';
+import { LinktaFlowRouter } from './routes/linktaFlowRouter';
 
 getEnv();
 const uri = process.env.MONGO_DB_URI;
+mongoose.set('strictQuery', false);
```

Comment:
Hey Reland, I tried running your branch and it's throwing on line 24 down below. 

```
Error: Missing DB connection string!
[1]     at startServer (/Users/colinvandergraaf/Documents/code/linkta/sandbox/server/server.ts:24:11)
[1]     at Object.<anonymous> (/Users/colinvandergraaf/Documents/code/linkta/sandbox/server/server.ts:127:1)
[1]     at Module._compile (node:internal/modules/cjs/loader:1368:14)
[1]     at Module.m._compile (/Users/colinvandergraaf/Documents/code/linkta/sandbox/node_modules/ts-node/src/index.ts:1618:23)
[1]     at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
[1]     at Object.require.extensions.<computed> [as .ts] (/Users/colinvandergraaf/Documents/code/linkta/sandbox/node_modules/ts-node/src/index.ts:1621:12)
[1]     at Module.load (node:internal/modules/cjs/loader:1205:32)
[1]     at Function.Module._load (node:internal/modules/cjs/loader:1021:12)
[1]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)
```
Am I supposed to a a connection string somewhere?

Reply:
Hi @colinvandergraaf, sorry I will provide you the connection string.